OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/ProviderControllerTest.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.Extensions.Logging;
```

Comment:
looks like we have 2 files ProviderControllerTest.cs 
could you check which one is not needed?